codegen/src/main/ops/org/nd4j/codegen/ops/Linalg.kt

```diff
@@ -206,6 +208,34 @@ fun Linalg() =  Namespace("Linalg") {
             """.trimIndent()
         }
     }
+
+    Op("tri") {
+        javaPackage = "org.nd4j.linalg.api.ops.custom"
+        javaOpClass = "Tri"
+
+        Arg(DataType.INT, "params") {count = AtLeast(3);"shapes of tensor and sub-diagonal number at and below which the array is filled"; }
```

Comment:
I think this should be `Between(1,3)` looking at the op. (i.e., 1 to 3 values, inclusive)
https://github.com/eclipse/deeplearning4j/blob/master/libnd4j/include/ops/declarable/generic/linalg/tri.cpp

So if iArgs are (x) then we get an [x,x] matrix
If iArgs are (x,y,) we get an [x,y] matrix
if iArgs are (x,y,z) we get an [x,y] matrix with the zth diagonal and below set to 1s